src/javascript/app/pages/cashier/payment_agent_withdraw.js

```diff
@@ -216,6 +216,9 @@ const PaymentAgentWithdraw = (() => {
                 break;
             default: // error
                 if (response.echo_req.dry_run === 1) {
+                    const $error = $('#form-error');
+                    $error.text(response.error.message);
+                    $error.removeClass('invisible');
                     setActiveView(view_ids.form);
                     $(field_ids.frm_msg).setVisibility(1).html(response.error.message);
```

Comment:
why doesn't this already work?

Reply:
Yep, there was a missing lookup value, I've changed it to use that. ended up removing these 3 lines.